t/t1210-pull.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# Copyright (c) 2006 Yann Dirson
```

Comment:
Copy/paste error?

Reply:
Removed, artifact of copying another test script.